app/process-batches/download-and-parse.js

```diff
@@ -5,7 +5,7 @@ const fileProcessingFailed = require('./file-processing-failed')
 
 async function downloadAndParse (filename, schemeType) {
   const buffer = await blobStorage.downloadPaymentFile(filename)
-  const parseSuccess = await parsePaymentFile(filename, buffer, schemeType.batchId)
+  const parseSuccess = await parsePaymentFile(filename, buffer, schemeType)
```

Comment:
Lower level functions within this need to know which scheme it is parsing so values are taken from correct location in file.  Therefore passing full `schemeType` object.